docs/reference/migration/migrate_7_0/plugins.asciidoc

```diff
@@ -22,6 +22,13 @@ must now be specified in the client settings instead.
 
 See {plugins}/repository-gcs-client.html#repository-gcs-client[Google Cloud Storage Client Settings].
 
+[float]
+==== S3 Repository Plugin
+
+* The plugin now uses the path style access pattern for all requests.
+In previous versions it was automatically determining whether to use virtual hosted style or path style
```

Comment:
Maybe `In previous versions the decision to use virtual hosted style or [..]  was automatically determined by the AWS Java SDK`